tests/functional/pykmip/ClusterClient.spec.js

```diff
@@ -0,0 +1,508 @@
+'use strict'; // eslint-disable-line strict
+
+const assert = require('assert');
+const { spawn } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const sinon = require('sinon');
+const util = require('util');
+
+const KmipClient = require('../../../lib/network/kmip/Client').default;
+const ClusterClient = require('../../../lib/network/kmip/ClusterClient').default;
+const {
+    logger,
+} = require('../../utils/kmip/ersatz');
+
+jest.setTimeout(5000);
```

Comment:
Please consider making the timeout configurable or environment-dependent.
In CI under load, 5s might not be enough and could cause flaky tests.

Reply:
The test is not in package.json, it's executed directly in its own CI workflow `yarn jest tests/functional/pykmip/ClusterClient.spec.js` separate from the other jest tests

So there is no other config to add, everything is self contained for this test

Reply:
In that case we should add it to package.json scripts as thats expected for our convention.

Reply:
added a script in package.json for pykmip tests